cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureDnsZoneService.java

```diff
@@ -106,10 +106,15 @@ public void getOrCreateDnsZones(AuthenticatedContext authenticatedContext, Azure
                     updateCloudResource(authenticatedContext, deploymentName, dnsZoneDeploymentId, CommonStatus.CREATED, AZURE_PRIVATE_DNS_ZONE);
 
                 }
-            } catch (CloudException | DataAccessException e) {
+            } catch (CloudConnectorException e) {
                 LOGGER.warn("Deployment {} failed due to {}", deploymentName, e.getMessage());
                 pollForCreation(authenticatedContext, azureClient, resourceGroup, deploymentName, dnsZoneDeploymentId,
```

Comment:
Just for clarification: why is poll for creation started when an exception is thrown? I have not found a catch clause in calling methods, I guess the creation will fail.

Reply:
The poll here is started to handle the status change and refresh the status of the resource to the up-to-date status seen in the cloud provider.